src/DependenciesPackager/Program.cs

```diff
@@ -164,10 +169,10 @@ static int Main(string[] args)
                 "Avoids deleting the package folders and the publish folder used for crossgen",
                 CommandOptionType.NoValue);
 
-            var runtimes = app.Option(
+            var runtime = app.Option(
```

Comment:
The JIT is various from platform to platform (year the actual term probably should be platform to be more accurate). We need to specify the value to extract correct JIT from package. In addition to use this string in file name to distinguish package caches for different platforms.